pontoon/base/static/js/translate.js

```diff
@@ -4371,6 +4371,12 @@ var Pontoon = (function (my) {
       } else {
         this.state = state;
       }
+
+      if (state.project == 'demo') {
+        setTimeout(function() {
+          Sideshow.start({ listAll: true });
+        }, 1000);
```

Comment:
Just to pass the condition of sideshow not defined. It can be solved if we keep translate.js in the last. but I still like the fading in transition due to 1 sec delay. I m removing the timeout in next commit though.